compiler/crates/schema/src/in_memory/mod.rs

```diff
@@ -1675,7 +1675,11 @@ impl InMemorySchema {
                     Ok(Argument {
                         name: ArgumentName(arg_def.name.value),
                         type_: self.build_input_object_reference(&arg_def.type_)?,
-                        default_value: arg_def.default_value.clone(),
+                        default_value: if let Some(default_value) = &arg_def.default_value {
+                            Some(default_value.value.clone())
+                        } else {
+                            None
+                        },
```

Comment:
Here (and the similar place in relay docblock) could you use `default_value.map(...)`? Might be a little cleaner.